rest/route/admin_test.go

```diff
@@ -175,9 +172,6 @@ func (s *AdminRouteSuite) TestAdminRoute() {
 	s.EqualValues(testSettings.LoggerConfig.Buffer.Count, settings.LoggerConfig.Buffer.Count)
 	s.EqualValues(testSettings.LoggerConfig.Buffer.UseAsync, settings.LoggerConfig.Buffer.UseAsync)
 	s.EqualValues(testSettings.LoggerConfig.Buffer.IncomingBufferFactor, settings.LoggerConfig.Buffer.IncomingBufferFactor)
-	s.EqualValues(testSettings.Notify.SMTP.From, settings.Notify.SMTP.From)
-	s.EqualValues(testSettings.Notify.SMTP.Port, settings.Notify.SMTP.Port)
-	s.Equal(len(testSettings.Notify.SMTP.AdminEmail), len(settings.Notify.SMTP.AdminEmail))
```

Comment:
Should we check that the SES settings are properly fetched?

Reply:
done